tutorials/fundamentals/left-trim.md

```diff
@@ -0,0 +1,20 @@
+Using a combination of JavaScript's `trimStart()`/`trimLeft()` function and a regular expression,
+you can clean up bad strings in your projects. By default, `trimStart`/`trimLeft()` will only remove
+whitespace from the start of the string. Therefore, you must define a function on the prototype of the
```

Comment:
Never ever recommend anyone define a property on `String.prototype` :)

Also, paragraph too long. Break it up into two paragraphs with 2 separate examples: first paragraph and first example uses plain old `trimStart()` to trim whitespace. 2nd paragraph and 2nd example uses a regexp to remove leading characters from a string, like `str.replace(/^(Na )+/, '')` removes all leading 'Na ' so 'Na Na Na Na Batman' becomes 'Batman'